domain/lib/src/model/sharedspacedocument/work_group_node_type.dart

```diff
@@ -30,7 +30,12 @@
 //  the Additional Terms applicable to LinShare software.
 //
 
-enum WorkGroupNodeType { FOLDER, DOCUMENT }
+import 'package:json_annotation/json_annotation.dart';
+
+enum WorkGroupNodeType {
+  @JsonValue('FOLDER') FOLDER,
```

Comment:
please don't use `@JsonValue` in `Domain` layer, it should be used only in `Data` layer. And we actually don't need to declare this for now IMO

Reply:
My bad I was debugging forgot to clean